data/lib/src/datasource_impl/document_datasource_impl.dart

```diff
@@ -127,4 +127,59 @@ class DocumentDataSourceImpl implements DocumentDataSource {
 
     return DownloadTaskId(taskId);
   }
+
+  @override
+  Future<Uri> downloadDocumentIOS(Document document, Token token, Uri baseUrl, CancelToken cancelToken) async {
```

Comment:
Why does download return an `Uri` please ?

Reply:
We need a downloaded filePath which is used to open share dialog (using this library [https://pub.dev/packages/share)](https://pub.dev/packages/share)